DEPS

```diff
@@ -458,6 +458,17 @@ deps = {
      'dep_type': 'cipd',
    },
 
+  'src/third_party/robolectric': {
+     'packages': [
+       {
+        'package': 'flutter/android/robolectric_bundle',
+        'version': 'Tox1em5FbLcoLvIIKfpMYH8lk_yLSvFsHxTa5cBjR-QC'
```

Comment:
Can we give this a more meaningful version tag?  Maybe something that corresponds with the version of RoboElectric in use.  These kinds of tags are really hard to figure out short of downloading the package and inspecting it manually.

Reply:
We could tag it based on Robolectric version, but then if somebody needed to add another runtime dependency (like Mockito) there'd be no way to increment it that made sense. This is just a collection of "robolectric+jars that we need to run it" so there isn't a version numbering scheme that actually makes sense for all of them at once really. That's why I went for just the opaque ID.

Maybe it would be better if each of the jars were all split out into separate packages? It would be about 20 of them, and I think need to do some magic to combine them all into a single directory for the `-Drobolectric.dependency.dir=` flag.

Reply:
Dan's makes a good point about the readability of those tags. Here is a potentially stupid idea. When I look at the DEPS file, I mostly just want to tell when they have last been updated. How about a tag like `updated:2019/08/22` or something like that for dependencies we own but don't have a good way to version (because of archive like these that are aggregated). For the specific version of the JARs, you could just put a readme in the archive stating the individual JAR versions.

Doesn't have to be this patch but food for thought.

Reply:
I went with Chinmay's suggestion to use `last_updated:<timestamp>` for now. I don't think it's technically infeasible to split these into separate packages, but I do think it would require some trade off to set up and manage that I'm not sure is worth it. You can tag CIPD versions with as many tags as you want, so I also added a `robolectric_version:3.8` tag just for extra obvious info on the Robolectric version specifically. Updated the testing README to talk about the versioning.